cmd/fyne_demo/widget.go

```diff
@@ -63,9 +63,10 @@ func Widget(app fyne.App) {
 	w := app.NewWindow("Widgets")
 
 	w.SetContent(widget.NewTabContainer(
+		widget.NewTabItem("Progress Tab 1", makeProgressTab()),
```

Comment:
Can this be rolled back too? It doesn’t add anything to what should be a widget demo window...

If we want to keep something in place for test purposes then it probably belongs in a unit test.